app/controllers/groups_controller.rb

```diff
@@ -63,6 +63,21 @@ def destroy
     end
   end
 
+  def transfer # rubocop:disable Metrics/AbcSize
+    new_namespace ||= Namespace.find_by(id: params.require(:new_namespace_id))
+    if Groups::TransferService.new(@group, current_user).execute(new_namespace)
+      flash[:success] = t('.success')
+      redirect_to group_path(@group)
+    else
+      @error = @group.errors.messages.values.flatten.first
+      respond_to do |format|
+        format.turbo_stream do
+          render status: :unprocessable_entity, locals: { confirm_value: @group.path, error: @error }
```

Comment:
**chore (blocking):** This is causing a full page refresh to update the error on the form.  You can take a look in the `main` branch at the `projects_controller#transfer` method and see how it is using `format.turbo_stream` to only return the form with error.

Reply:
My mistake, I did not have the latest code pulled.  This works great, thanks @ksierks